### PR TITLE
cypress-guerrillamail

### PR DESCRIPTION
**Pull request by e23thr**
A Cypress plugin to generate random email, using [guerrillamail.com](https://www.guerrillamail.com/) and read its mail box that could be used for automated testing.

### CODE REVIEW

File: src/data/plugins.json [hunk 1]
```diff
@@ -1227,6 +1227,13 @@
           "link": "https://github.com/SMenigat/cypress-mailhog",
           "keywords": ["email", "mailhog", "test", "commands"],
           "badge": "community"
+        },
+        {
+          "name": "cypress-guerrillamail",
+          "description": "Commands to get random email address, using guerrillamail.com and read its mail box that could be used for automated testing.",
```
Comment (mike-plummer) [inline src/data/plugins.json:1233]:
```suggestion
          "description": "Create and use a randomly-generated email address from Guerrilla Mail.",
```

**Review by mike-plummer — CHANGES_REQUESTED** [review]
Thanks for the contribution! One small suggestion on the wording of the description, let me know what you think

**Comment by e23thr** [comment]
Yes, thanks

File: src/data/plugins.json (hunk 1 above)
Comment (e23thr) [reply src/data/plugins.json:1233]: @mike-plummer  Thanks for your suggestions. Do I have to update in my repo first? Do I have to resync? I'm new to contributions.

**Comment by e23thr** [comment]
Thanks @mike-plummer for your kindness and quick action

**Pull request merged**